diff --git a/boards/nucleo-l073rz/include/periph_conf.h b/boards/nucleo-l073rz/include/periph_conf.h
index 54adcabf02..0f259905f0 100644
--- a/boards/nucleo-l073rz/include/periph_conf.h
+++ b/boards/nucleo-l073rz/include/periph_conf.h
@@ -143,7 +143,13 @@ static const adc_conf_t adc_config[] = {
     { GPIO_PIN(PORT_A, 4), 4 },
     { GPIO_PIN(PORT_B, 0), 8 },
     { GPIO_PIN(PORT_C, 1), 11 },
-    { GPIO_PIN(PORT_C, 0), 10 }
+    { GPIO_PIN(PORT_C, 0), 10 },
+    /* ADC VLCD channel */       \
+    { GPIO_UNDEF, 16 },          \
+    /* ADC VREF channel */       \
+    { GPIO_UNDEF, 17 },          \
+    /* ADC Temperature channel */\
+    { GPIO_UNDEF, 18 }
 };
 
 #define ADC_NUMOF           ARRAY_SIZE(adc_config)
